shared/postgres/scripts/insert_data.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+cleanup() {
+  echo "Interrupted. Exiting..."
```

Comment:
Beware that anything echoed to stdout will be a normal `INFO` log, if you want to produce an `ERROR` you need to write it to stdout (for instance, this is how we do it in the [mongo container](https://github.com/ConduitIO/streaming-benchmarks/blob/288464f3f84c9ffe79fc3d9a57679c71e548c235/shared/mongo/scripts/common.sh#L17C1-L17C30)).